index.html

```diff
@@ -1,41 +1,191 @@
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="about-wrapper">
+            <div class="about-me">
+                <h1>About me</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+                <a href="#">Get My Resume</a>
+                <!-- <img id="about-bg" src="/assets/images/Group.png"/> -->
+            </div>
+            <div class="about">
+                <hr>
+                <div class="title">
+                    <img src="/assets/images/kite.png" alt="img">
+                    <h1>Language</h1>
+                </div>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+                <div class="title">
```

Comment:
Good job for using ul/li tag .
For the same reason, you should use it for Language/framework/skills